mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -216,7 +216,7 @@ func Provider() *schema.Provider {
 			"google_compute_backend_service":                   dataSourceGoogleComputeBackendService(),
 			"google_compute_backend_bucket":                    dataSourceGoogleComputeBackendBucket(),
 			"google_compute_default_service_account":           dataSourceGoogleComputeDefaultServiceAccount(),
-			"google_compute_disk":        					    dataSourceGoogleComputeDisk(),
+			"google_compute_disk":        		            dataSourceGoogleComputeDisk(),
```

Comment:
Same comment around whitespace changes: could you revert them here and on the other lines in this file?